submissions/natalia-artsiukh/html-css-popup/style.css

```diff
@@ -0,0 +1,149 @@
+body {
+    margin: 0;
+    padding: 0;
+    background-image: url(./img/ocean.jpg);
+    background-repeat: no-repeat;
+    box-sizing: border-box;
+}
+.header {
+    background-color: rgba(110, 120, 116, 0.598);
+}
+.header_nav {
+    display: flex;
+    justify-content: end; 
+}
+.nav-position {
+    display: flex;
+    justify-content: space-evenly;
+    width: 30%;
+    align-items: center;
+    line-height: 3em;
+}
+.link {
+    text-decoration: none;
+    color: blanchedalmond;
+    font-size: 18px;
```

Comment:
To use a responsive design, even when a page will be scaled you can use `em` or `rem`